recipes/niftyreg/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "niftyreg" %}
+# Versioning in this project is tricky.
+# - The last tag is out of date
+# - Their CMake config says 1.5.<number on niftyreg_build_version.txt>
+# - That file is controlled by a git hook
+# - There have been commits to master since the last change in that file though
+# - I'll add a 4th component: changes to master since last modification of niftyreg_build_version.txt
```

Comment:
:unamused:

Reply:
Bad idea? Are there any alternatives that we should consider instead?

Reply:
Oh. I'm not complaining about your idea but rather about the bad state of this package metadata.

Reply:
Ah, ok hah :D I misunderstood then. Yes... this one is fun